server/shared/publishing/helpers.js

```diff
@@ -143,72 +138,62 @@ function publishAssessments(parent) {
 }
 
 function fetchContainers(repository, parent) {
-  const config = find(repository.getSchemaConfig().structure, pick(parent, 'type'));
-  const containerTypes = get(config, 'contentContainers', []);
-  return parent.getChildren({ where: { type: containerTypes } })
-    .then(containers => Promise.map(containers, fetchContainer));
+  const schemaConfig = repository.getSchemaConfig();
```

Comment:
Try to extract some part / parts into method to make this one bit more readable...